app/(main)/register/page.test.tsx

```diff
@@ -170,4 +173,31 @@ describe('Register', () => {
 
     expect(darkModeSection).toHaveClass('dark:bg-gradient-to-b');
   });
+
+  test('Should show loadingspinner in submit button when clicked', async () => {
```

Comment:
We still need another test that checks the submit fired, and the loader is not there anymore.